### PR TITLE
Activate "Authorised IPs" page for all PrivateSQL

### PR DESCRIPTION
**Pull request by wdhif**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | PUDBLOW-1330
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

* Activate "Authorised IPs" page for all PrivateSQL
* Convert PrivateSQL to CloudDB

### CODE REVIEW

**Comment by wdhif** [comment]
PR has internal blockers for now.

**Review by mohammed-zahaf — DISMISSED** [review]

**Comment by wdhif** [comment]
PR is ready for review. Internal blockers are fixed.

**Comment by JeremyDec** [comment]
Hello @wdhif,

Your PR still have an issue with its CDS build, you might need to rebase your branch. Can you check it please?

Thanks

**Comment by wdhif** [comment]
@JeremyDec PR rebased 😄

File: packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_de_DE.json [hunk 1]
```diff
@@ -470,7 +469,7 @@
   "privateDatabase_modale_oom_tab_size_reached": "Erreichte Kapazität",
   "privateDatabase_modale_oom_tab_uptime": "Zeit seit der letzten Überschreitung",
   "privateDatabase_modale_oom_no_oom": "Keine Überschreitung der RAM-Kapazität.",
-  "privateDatabase_modale_oom_info_order": "Wir empfehlen Ihnen, die RAM-Kapazität Ihrer {{t0}} zu erhöhen, um diese Probleme zu beheben.",
+  "privateDatabase_modale_oom_info_order": "Wir empfehlen Ihnen, die RAM-Kapazität Ihrer CloudDB zu erhöhen, um diese Probleme zu beheben.",
```
Comment (marie-j) [inline packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_de_DE.json:472]: According to https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations only `Messages_fr_FR.json` should be updated

**Review by marie-j — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_de_DE.json (hunk 1 above)
Comment (wdhif) [reply packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_de_DE.json:472]: Seen together, not critical for this one.

**Pull request merged**